src/containers/App.tsx

```diff
@@ -1,340 +1,92 @@
-          </div>
-          <div className="flex flex-row w-[150px] m-1">
-            <PersistentInput name={REFETCH_ON_MOUNT} />
-          </div>
-        </div>
-        <div className="flex flex-row justify-start">
-          <ThemeSelector />
-        </div>
-      </div>
-      {showServerEvents && (
-        <div className="flex m-2 card bg-base-100 shadow-xl">
-          <div className="card-body">
-            <div className="flex flex-row">
-              <span className="card-title">Server events</span>
-              <button
-                className={`btn btn-sm btn-success m-1`}
-                disabled={serverEventsState === "connected"}
+    <div className="flex flex-col w-full-no-scrollbar h-full box-border pt-4">
+      <div className="tabs tabs-boxed m-2">
+        {state.rooms === null && <div>...</div>}
+        {Object.values(state.rooms || {}).map((room) => {
+          return (
+            <div key={room.id} className="indicator">
+              <CloseButton
+                position={"left"}
                 onClick={() => {
-                  if (!serverMessagesWebsocket) {
-                    showToastError("serverMessagesWebsocket websocket is null");
-                    return;
-                  }
-
-                  const ws = new WebSocket(serverMessagesWebsocket);
-                  const handler = (event: unknown) => {
-                    if (event instanceof MessageEvent) {
-                      const newData = JSON.parse(event.data);
-                      setServerMessages((prevState) => [
-                        ...prevState,
-                        {
-                          data: newData,
-                          id: crypto.randomUUID(),
-                        },
-                      ]);
-                    }
-                  };
-                  ws.addEventListener("message", handler);
-                  ws.addEventListener("close", () => {
-                    console.warn("close");
-                  });
-                  ws.addEventListener("error", () => {
-                    console.warn("error");
-                  });
-
-                  ws.addEventListener("open", () => {
-                    console.log("Opened!");
-                    setServerEventsState("connected");
-                    ws.send(
-                      JSON.stringify({
-                        type: "controlMessage",
-                        data: { type: "authRequest", token: serverToken },
-                      }),
-                    );
+                  roomApi?.jellyfishWebRoomControllerDelete(room.id).then(() => {
+                    const LOCAL_STORAGE_KEY = `tokenList-${room.id}`;
+                    removeSavedItem(LOCAL_STORAGE_KEY);
+                    refetchRoomsIfNeeded();
                   });
                 }}
+              />
+              <a
+                className={`tab tab-lifted tab-lg ${state.selectedRoom === room.id ? "tab-active" : ""}`}
```

Comment:
Instead of bugfixing i replaced tab-lifted with tab-bordered. No artefact and it looks even better!
![image](https://github.com/jellyfish-dev/jellyfish-dashboard/assets/42720407/bdabc88c-8bfe-4c0c-8efa-4ed0bec24247)